google/cloud/spanner/samples/samples.cc

```diff
@@ -3885,9 +3973,51 @@ void RunAll(bool emulator) {
   std::cout << "\nRunning the make-delete-mutation example" << std::endl;
   MakeDeleteMutation(client);
 
+  if (!emulator) {
+    std::cout << "\nRunning spanner_query_information_schema_database_options"
+              << " sample" << std::endl;
+    QueryInformationSchemaDatabaseOptions(client);
+  }
+
   std::cout << "\nRunning spanner_drop_database sample" << std::endl;
   DeleteAll(client);
   DropDatabase(database_admin_client, project_id, instance_id, database_id);
+
+  // TODO(#7144): Awaiting emulator support for default_leader.
+  if (!emulator) {
```

Comment:
For a future PR, the emulator keeps falling behind.  Maybe the CI builds should run some of the integration tests against production too?

Reply:
I'm afraid you're preaching to the choir on that one.  When I've suggested it, I've lost.  I guess we still have "integration-daily", but I certainly edit `google/cloud/spanner/ci/run_integration_tests_emulator_bazel.sh` when I'm developing spanner stuff so that I get some pre-merge prod coverage.